bin/statique

```diff
@@ -48,13 +48,10 @@ Http.createServer(function (req, res) {
     let p = Abs(req.url);
     // check if it exists in the file system
     fs.exists(p, function (exists) {
```

Comment:
You can replace `fs.exists` with `fs.stat` so, that you get the stat in the callback (and the first argument in the callback would be the error).